JavaScript - ESS6 Modules: Change verbiage on imports/exports (#25987)
TheOdinProject/curriculum: javascript/organizing_your_javascript_code/es6_modules.md

Now that we (sorta) understand what webpack is doing it's time to discuss the module syntax. There are only 2 components to it - __import__ and __export__.

> - Take a moment to look at the [MDN documentation about the import deceleration](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import) and the [MDN documentation about the export deceleration](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export) which give great examples for the different syntax on import and export.

Author: Ha I knew those did not look right, but my poor spelling head could not grasp it.  Thank you!

> - Take a moment to look at the [MDN documentation about the import deceleration](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import) and the [MDN documentation about the export deceleration](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export) which give great examples for the different syntax on import and export.

Reviewer: Would it be possible to frame this in a "softer" manner? Learners may consider this to be a "must-read" and get demotivated/ be unsure about having to read the full documentation on imports/exports without a simpler introduction, first.

Reviewer: In general, the issue with the specific documentation at the beginning of the lesson's section on these statements may be a deterrent, which's why they were *specifically* mentioned at the end. While the previous arrangement may not be perfect - can you think of anything better?

Author: Actually they were linked in both places, but I have some ideas to help.  I will work on it when back at computer

Author: I added a new commit, what do you think?

Reviewer: LGTM! I like it. Feel free to close this, I love the new version. No stray links, and very to-the-point.

Now that we (sorta) understand what webpack is doing it's time to discuss the module syntax. There are only 2 components to it - __import__ and __export__.

> The import statement is the same thing that you used during the webpack tutorial! These things are simple to use.

Reviewer 2: ```suggestion
The import statement is the same thing that you used during the webpack tutorial so it should be familiar by now.
```
nit: in case someone doesn't find this simple :)

Easy! Now, if you run `npx webpack` in your project directory, your page should show our new function being used.

<span id="exports-knowledge-check"></span> 

> There are two different ways to use exports in your code: named exports and default exports. Which option you use depends on what you're exporting.vTake a moment to look at the [MDN documentation about the import declaration](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import) and the [MDN documentation about the export declaration](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export) which give great examples for the different syntax on import and export.

Reviewer: ```suggestion
There are two different ways to use exports in your code: named exports and default exports. Which option you use depends on what you're exporting. Take a moment to look at the [MDN documentation about the import declaration](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import) and the [MDN documentation about the export declaration](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export) which give great examples for the different syntax on import and export.
```

> Here is an example of use named exports which you will most often use when you have multiple values in a module to export:

Reviewer: ```suggestion
Here is an example of with named exports, which you will most often use when you have multiple values to export in a module:
```
A slight rephrasing, since one may want to stress on the fact that you are "Exporting multiple values"

- [Describe how to write an npm automation script.](https://peterxjang.com/blog/modern-javascript-explained-for-dinosaurs.html)
- [Explain one of the main benefits of writing code in modules.](#module-knowledge-check)
- [Explain "named exports" and "default exports".](#exports-knowledge-check)

### Additional resources

This section contains helpful links to other content. It isn’t required, so consider it supplemental.

> - [A video about ESS6 Modules](https://youtu.be/cRHQNNcYf6s) if you find video lessons easier to absorb.

Reviewer 2: ```suggestion
- [Watch this video about ESS6 Modules by Web Dev Simplified](https://youtu.be/cRHQNNcYf6s) if you find video lessons easier to absorb. It covers the same topics as discussed in this lesson.
```

I credited the channel in the link and thought it important to note that no extra topics were covered. If you feel there are extra topics covered, please let me know. I am also open to your feedback here of course.
